Project.toml

```
@@ -1,23 +1,26 @@
 name = "HistogramThresholding"
 uuid = "2c695a8d-9458-5d45-9878-1b8a99cf7853"
-authors = ["Dr. Zygmunt L. Szpak <zygmunt.szpak@gmail.com>"]
-version = "0.2.5"
+authors = ["Dr. Zygmunt L. Szpak <zygmunt.szpak@gmail.com> and contributors."]
```

I think usually we just list the leading/original authors here.

The `authors` is actually a list so it should be multiple strings (the [TOML format](https://toml.io/en/)):

```suggestion
authors = ["Dr. Zygmunt L. Szpak <zygmunt.szpak@gmail.com>", "JuliaImages"]
```

for example, https://github.com/JuliaTesting/ReferenceTests.jl/blob/fd8949f4ba648aa31b8321163a54aa076d65bd82/Project.toml#L3